### DISCUSSION TITLE
opendoc/start/boot/%E5%90%8E%E7%AB%AF%E5%90%AF%E5%8A%A8

### DISCUSSION THREAD

**Discussion by giscus** [zuihou/zuihou-admin-cloud#348] [企业版]
# opendoc/start/boot/%E5%90%8E%E7%AB%AF%E5%90%AF%E5%8A%A8

微服务中后台快速开发平台

https://tangyh.top/opendoc/start/boot/%E5%90%8E%E7%AB%AF%E5%90%AF%E5%8A%A8.html

**Comment by yyxhgy** [comment]
单体版后端启动后，访问http://127.0.0.1:18760/doc.html ，切换oauth文档后，服务报错Handler dispatch failed，这个是什么原因？拿的是gitee 4.x_java17最新的代码

**Reply by yyxhgy** [reply]
我的理解，单体版none模式启动，应该是不需要nacos的吧？我启动了前后端项目后访问登录页，验证码图片是404，我断点调试了发现请求是到后端的，但是没进具体的controller

**Reply by zuihou** [reply]
文档： https://tangyh.top/opendoc/start/boot/%E5%89%8D%E7%AB%AF%E5%90%AF%E5%8A%A8.html

后端是lamp-boot，前端启动lamp-web时，需要手动修改配置：

修改 .env.development：
```
VITE_PROXY=[[["/api/gateway", "/api/gateway", "/gateway"],"http://localhost:18760"], [["/api", "/api/[A-Za-z0-9]+", ""],"http://localhost:18760"],["/basic-api","http://localhost:3000"],["/upload","http://localhost:3300/upload"]]
```

**Reply by zuihou** [reply]
swagger 问题，更新一下lamp-boot代码，漏提了一个类

**Reply by yyxhgy** [reply]
感谢解惑

**Comment by bpg12333** [comment]
请问一下java: 找不到符号 符号:   类 Serial 位置: 程序包 java.io  但是jdk我是17对比了得

**Reply by zuihou** [reply]
idea 的所有设置jdk的地方，都统一设置为 jdk17，不清楚的地方，百度

**Reply by bpg12333** [reply]
这个我处理  为啥我单体项目启动显示链接不了数据库  我已经按照文档步骤全部操作了一遍了启动得时候出现数据源没有呀

**Reply by bpg12333** [reply]
谢谢已经弄好了
